MIGRATIONS.md

```diff
@@ -3,6 +3,9 @@
 This file documents changes in the data model. Please explain any changes to the
 data model as well as any custom migrations.
 
+## WordPress 49 (@jleandroperez 2016-05-04)
```

Comment:
I'd suggest following the format for WordPress 48 if we do multiple edits just so its clear the demarcation of each section is model version number.